features/step_definitions/journey/registration_steps.rb

```diff
@@ -17,6 +17,20 @@
   @tier = "upper"
 end
 
+Given(/^a limited company with companies house number "([^"]*)" is registered as an upper tier waste carrier$/) do |ch_no|
+  # Store variables for later steps:
+  @business_name = "AD UT Company convictions check ltd"
+  @companies_house_number = ch_no
+
+  step("I want to register as an upper tier carrier")
+  step("I start a new registration journey in 'England' as a 'limitedCompany'")
+  step("I complete my registration")
+  step("I pay by card")
```

Comment:
I know we discussed about re-using helper rather than steps, but in this case, this just seemed better. I can create a PR where I refactor the code inside these steps to be in helpers and call the helpers if you like to see the difference and then we can decide what's better if you like :) won't take long to do

Reply:
I agree this makes sense in this case. I like the idea of condensing regularly-used combinations of steps into a single one, so that the feature is easier to read. Let's go with this!